roles/adoption_osp_deploy/README.md

```diff
@@ -0,0 +1,34 @@
+# adoption_osp_deploy
+
+Deploy OSP 17.1 environment for adoption based on DTs.
+
+## Privilege escalation
+None
+
+## Parameters
+* `cifmw_adoption_osp_deploy_ntp_server`: (String) NTP server to use in the 17.1
+deployment. Defaults to `pool.ntp.org`
+* `cifmw_adoption_osp_deploy_repos`: (List) List of 17.1 repos to enable. Defaults to
+`[rhel-9-for-x86_64-baseos-eus-rpms, rhel-9-for-x86_64-appstream-eus-rpms, rhel-9-for-x86_64-highavailability-eus-rpms, openstack-17.1-for-rhel-9-x86_64-rpms, fast-datapath-for-rhel-9-x86_64-rpms, rhceph-6-tools-for-rhel-9-x86_64-rpms]`
+* `cifmw_adoption_osp_deploy_skip_stages`: (String or List) Stages to skip
```

Comment:
hmm I'd use actual `tags: ['undercloud', 'overcloud']` and leverage `--skip-tags`. That would be better, since it would really skip the tasks without even having to consider the actual tasks with some `when` condition.

UNLESS this would be needed in CI context? Not sure if Zuul job definition allows to pass down and tags to skip... ?

Reply:
I can't think of any use case for this in a CI job, I added it mostly to avoid having to re-run the undercloud deployment whenever something went wrong with the overcloud deploy. I remembered this was implemented in kustomize_deploy and copied the idea from there. But you're right that this could be much simpler with tags, I'll give it a try locally and update the PR

Reply:
I've updated the PR moving to ansible tags, it's indeed simpler, thanks @cjeanner